### PR TITLE
Add morphology based neuron classes

### PR DESCRIPTION
**Pull request by shawntanzk**
Added the following: 
Bitufted neuron 
Martinotti neuron
Chandelier cell

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -28157,6 +28158,24 @@ SubClassOf(obo:CL_4023071 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023071 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P09240))
 
 
+# Class: obo:CL_4023076 (Martinotti neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Martinotti_cell&oldid=9947732") obo:IAO_0000115 obo:CL_4023076 "An interneuron that has Martinotti neuron. These interneurons are scattered throughout various layers of the cerebral cortex, sending their axons up to the cortical layer I where they form axonal arborization.")
```
Comment (patrick-lloyd-ray) [inline src/ontology/cl-edit.owl:28163]: Seems like a typo. Perhaps:

"An interneuron that has Martinotti morphology."

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -28157,6 +28158,24 @@ SubClassOf(obo:CL_4023071 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
...
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023076 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023076 "ILX:0107375")
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023076 "nifext_55")
+AnnotationAssertion(rdfs:label obo:CL_4023076 "Martinotti neuron")
+EquivalentClasses(obo:CL_4023076 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070007)))
+
+# Class: obo:CL_4023077 (bitufted neuron)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023077 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:18568015") oboInOwl:hasDbXref obo:CL_4023077 "A type of interneuron that has two clusters of branches that originate directly from the soma and extend in opposite directions and axons that form a plexus which spreads widely. Compared to bipolar neurons, bitufted neurons have branching that occur close to the soma.")
```
Comment (patrick-lloyd-ray) [inline src/ontology/cl-edit.owl:28173]: A type of interneuron that has two clusters of _dedritic_ branches? Or is it unambiguous that all branches-talk is about dendrites?

**Review by patrick-lloyd-ray — COMMENTED** [review]
inline

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:28173]: dendritic is right, took this from the Petilla terminology which and dendrites as the subtitle hence them not adding that word in the text, thanks for spotting it :)

**Review by dosumis — APPROVED** [review]

**Pull request merged**